src/app/login/page.tsx

```diff
@@ -0,0 +1,83 @@
+"use client";
+import { useState } from "react";
+import Footer from "@/components/Footer";
+import Login from "@/components/Login";
+import Link from "next/link";
+import { signInWithGoogle } from "@/lib/firebase/auth";
+import AuthLayout from "../authLayout";
+
+export default function LoginPage() {
+  const [openModal, setOpenModal] = useState(false);
+
+  const handleModal = () => {
+    setOpenModal(!openModal);
+  };
+
+  return (
+    <>
+      {openModal ? <Login handleModal={handleModal} /> : null}
+      <main className="flex min-h-screen flex-col items-center justify-between">
+        <section className="text-gray-400 body-font">
+          <div className="container flex px-5 py-24 md:flex-row flex-col items-center">
+            <div className="lg:max-w-lg lg:w-full md:w-1/2 w-5/6 md:mb-0 mb-10 flex flex-col flex-grow items-center">
+              <img
+                className="h-120 object-center rounded mb-4"
+                alt="Social Verse Logo"
+                src="/images/logo_main.png"
+              />
+            </div>
+            <div className="md:w-1/2 lg:pl-24 md:pl-16 flex flex-col md:text-left items-center space-y-4">
+            <h1 className="text-3xl font-bold text-white mb-4">
+                Join Social Verse
+              </h1>
+              <button
+                onClick={() => signInWithGoogle()}
+                className="px-14 w-full py-2 border flex gap-2 bg-blue-500 border-blue-500 rounded-full text-white hover:border-white hover:text-white hover:shadow transition duration-150 items-center justify-center"
```

Comment:
the `bg-blue-500` blends in with the google icon, maybe try a different shade of blue